README.org

```diff
@@ -1,17 +1,105 @@
 #+TITLE: Ren'Py mode
 
-PyTom's old major mode for Ren'Py, previously maintained by treymerkley, now maintained by me.
+A major mode for editing Ren'Py files.
 
 ** Installation
 
-  Either install within Emacs directly from MELPA or:
+Either install the ~renpy~ package from within Emacs directly from MELPA or:
+
   * Download/clone this repo to your desired location.
-  * In Emacs, use the command =M-x package-install-file= and point it to the renpy.el file location
-    * This will install it to your =~/.emacs.d/elpa/= directory (or your custom location if you've manually set a different =package-user-dir= in Emacs)
-  * In your .emacs file, add =(require 'renpy)=
+
+  * In Emacs, use the ~package-install-file~ command and point it to the renpy.el file
+    location
+
+** Features
+
+  * Movement, indentation and syntax highlighting for Ren'py and embedded Python.
+
+  * Outline Minor Mode support for an overview of block-defining statements..
+
+  * In-buffer navigation with Imenu.
+
+  * Context-sensitive symbol completion.
+
+  * Jumping to symbol definitions with Xref.
+
+  * Compiling, running, linting from within Emacs.
+
+  * Running renpy CLI subcommands.
+
+  * Error highlighting with Flymake.
+
+** Configuration
+
+Example configuration with the ~(use-package)~ macro:
+
+#+begin_src emacs-lisp
+  ;; Replace with the path to renpy and the location of renpy.el if
+  ;; needed.
+  (use-package renpy
+    ;; Use MELPA package, omit if you installed manually
+    :ensure t
+    :custom
+    ;; Path to the Ren'Py launcher or renpy.sh / renpy.exe.
+    (renpy-program "path/to/renpy")
+    ;; Enable context-sensitive completion (capf).
+    (renpy-setup-completion t)
+    :bind
+    ;; Bind some useful commands.
+    ("C-c C-c" . renpy-compile)
+    ("C-c C-r" . renpy-run)
+    ("C-c C-l" . renpy-lint)
+    :hook
+    ;; Enable on-the-fly linting and code overview
+    ((renpy-mode . flymake-mode)
+     (renpy-mode . outline-minor-mode))) ; code overview
+#+end_src
+
+If you installed the single ~renpy.el~ file manually, make sure it is on ~load-path~ and
+then:
```

Comment:
I think it needs to be made clear exactly what is meant by "installing manually", since `(require 'renpy)` is not required for the two installation methods which are described above, but someone might assume that using `package-install-file` is the "manual" method.